src/heartbeat.ts

```diff
@@ -76,8 +76,13 @@ export class HeartbeatManager extends Disposable {
         this._register(vscode.window.onDidCloseTerminal(() => this.updateLastActivity('onDidCloseTerminal')));
         this._register(vscode.window.onDidChangeTerminalState(() => this.updateLastActivity('onDidChangeTerminalState')));
         this._register(vscode.window.onDidChangeWindowState((e) => {
-            this.focused = e.focused;
-            this.updateLastActivity('onDidChangeWindowState');
+            // active property was introduced in 1.89, but not exist in 1.75
```

Comment:
You need to increase `"engines": {"vscode": "^1.82.0"},` in package.json and also the vscode types in devDependencies

Reply:
It requires `@types/vscode` version `1.89`, then our extension will need to update engines to `^1.89.0`

Reply:
Resolve as we don't need to check `active` anymore